doc/source/whatsnew/v2.2.2.rst

```diff
@@ -21,7 +21,8 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+-- Fixed bug in :func:`pandas.io.json_normalize` raising with errors='ignore' while traversing empty list (:issue:`57810`)
```

Comment:
Minor but you need just one hyphen for this to render properly

Reply:
Related to comment below on test I also don't think we should mention the `errors="ignore"` argument here as that is unrelated to the change